service/policy/db/resource_mapping.go

```diff
@@ -48,17 +54,29 @@ func (c PolicyDBClient) GetResourceMappingGroup(ctx context.Context, id string)
 		return nil, db.WrapIfKnownInvalidQueryErr(err)
 	}
 
+	metadata := new(common.Metadata)
+	if err := unmarshalMetadata(rmGroup.Metadata, metadata, c.logger); err != nil {
+		return nil, err
+	}
+
 	return &policy.ResourceMappingGroup{
 		Id:          rmGroup.ID,
 		NamespaceId: rmGroup.NamespaceID,
 		Name:        rmGroup.Name,
+		Metadata:    metadata,
 	}, nil
 }
 
 func (c PolicyDBClient) CreateResourceMappingGroup(ctx context.Context, r *resourcemapping.CreateResourceMappingGroupRequest) (*policy.ResourceMappingGroup, error) {
+	metadataBytes, _, err := db.MarshalCreateMetadata(r.GetMetadata())
```

Comment:
It looks like this would marshal JSONB with something like `{"labels":{}}` and an empty map when the metadata was empty on create. Perhaps we should check that behavior, and if that's what it indeed does, check if it's nil and pass nil bytes in that case?